### ISSUE TITLE
Cannot focus the terminal after tabbing out of the terminal accessibility buffer

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#173071]
https://github.com/xtermjs/xterm.js/issues/4394

1. `Shift+Enter` to enter accessibility buffer
2. `Tab` to exit
3. Try to click in the terminal 🐛 it won't focus

**Comment by aiday-mar** [comment]
Verified, if I am not mistaken now the command to enter the accessibility buffer is `shift+tab`
